src/test/java/com/icthh/xm/tmf/ms/communication/web/rest/EmailTemplateControllerTest.java

```diff
@@ -119,6 +153,22 @@ public void getEmailSpec() {
         verifyNoMoreInteractions(emailSpecService);
     }
 
+    @Test
+    @SneakyThrows
+    public void testUpdateTemplate() {
+        UpdateTemplateRequest updateTemplateRequest = createUpdateRequestTemplate();
+
+        String templateKey = "templateKey1";
```

Comment:
Please make templateKey - constant because we will use it in the next test cases

Reply:
done